package.json

```diff
@@ -45,6 +45,7 @@
     "react-dom": "^0.14.0",
     "react-transform-catch-errors": "^1.0.2",
     "react-transform-hmr": "^1.0.2",
+    "prop-types": "^15.0.0",
```

Comment:
This is a different version that waht you have in Peer Dependencies. The maintainers recommend that `15.5.7` is the minimum version you should use:

https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package

Reply:
@GGGGino can you update this, and then I will merge. Thank you!

Reply:
Pushed. Give a look